util/topgen/top_uvm_reg.sv.tpl

```diff
@@ -41,6 +41,14 @@
       reg_block_path = reg_block_path if block.hier_path is None else block.hier_path + "." + reg_block_path
 %>\
 // Block: ${block_name.lower()}${if_desc}
+<%
```

Comment:
Please file a GitHub issue explaining this hack. Please add comment here as well:
```
  // TODO(#BUG-NUMBER): <Explanation for this hack.>.

Reply:
Thanks Sri. Filed issue #8204 and added a comment in the file.